### DISCUSSION TITLE
[Firebase Analytics] Android - The parameter value in GA 360 for logEvent displays firebase_error: 4 when the parameter value is less than 500 characters.

### DISCUSSION THREAD

**Discussion by abenavent96** [capawesome-team/capacitor-firebase#600] [Q&A]
I put firebaseAnalyticsVersion to 21.5.1 into variables.gradle to get the new feature:
![Captura de pantalla 2024-03-27 a las 14 06 37](https://github.com/capawesome-team/capacitor-firebase/assets/42931161/52c2d1b6-2ebf-474f-83de-7daf553e9953)

When I send a log event with a parameter value length(< 500 characters), it displays the firebase_error: 4 .
![Captura de pantalla 2024-03-27 a las 14 08 08](https://github.com/capawesome-team/capacitor-firebase/assets/42931161/7449cad9-bde8-4ec1-bdfe-0d46e830c34e)

**Comment by robingenz** [comment]
Make sure to sync your Gradle files after setting the variable.

**Reply by abenavent96** [reply]
@robingenz I already did it and the problem persists. Could be an issue?
